testsuite/gnat2goto/tests/ranges/ranges.adb

```diff
@@ -3,9 +3,13 @@ procedure Ranges is
    type Int2 is new Int1 range 2 .. 9;
    subtype Int3 is Int1 range 3 .. 8;
```

Comment:
The test should make sure that the integer types here genuinely have different sizes (in bits), not just different ranges